static-site/src/components/splash/index.tsx

```diff
@@ -1,135 +1,127 @@
+          imgSrc={require("../../../static/logos/fa-viruses-solid.svg").default.src}
+          title="Core pathogens"
+          abstract="Continually updated views of a range of pathogens maintained by the Nextstrain team"
+          link="/pathogens"
         />
         <Section
-          id="groups"
-          title="Nextstrain Groups"
-          abstract="We want to enable research labs, public health entities and others to share their datasets and narratives through Nextstrain with complete control of their data and audience."
-          buttonText="See all groups"
-          buttonLink="/groups"
+          id="sars-cov-2"
+          // eslint-disable-next-line @typescript-eslint/no-var-requires
+          imgSrc={require("../../../static/logos/fa-virus-covid-solid.svg").default.src}
+          title="SARS-CoV-2"
+          abstract="Up-to-date analyses and a range of resources for SARS-CoV-2, the virus responsible for COVID-19 disease"
+          link="/sars-cov-2"
         />
         <Section
-          id="pathogens"
-          title="Explore pathogens"
-          abstract="Genomic analyses of specific pathogens kept up-to-date by the Nextstrain team."
-          buttonText="See all pathogens"
-          buttonLink="/pathogens"
+          id="tooling"
+          // eslint-disable-next-line @typescript-eslint/no-var-requires
+          imgSrc={require("../../../static/logos/fa-screwdriver-wrench-solid.svg").default.src}
+          title="Open source tooling"
+          abstract="Bioinformatic workflows, analysis tools and visualization apps for use by the community"
+          link="https://docs.nextstrain.org/en/latest/install.html"
         />
         <Section
-          id="community"
-          title="From the community"
-          abstract={(<>
-            Analyses by independent groups <a href="https://docs.nextstrain.org/en/latest/guides/share/community-builds.html">stored and
-            accessed via public GitHub repos</a>
-          </>)}
-          buttonText="Learn more"
-          buttonLink="/community"
+          id="nextclade"
+          // eslint-disable-next-line @typescript-eslint/no-var-requires
+          imgSrc={require("../../../static/logos/nextclade-logo.svg").default.src}
+          title="Nextclade"
+          abstract="In-browser phylogenetic placement, clade assignment, mutation calling and sequence quality checks"
+          link="https://clades.nextstrain.org"
         />
         <Section
-          id="narratives"
-          title="Narratives"
-          abstract="Narratives are a method of data-driven storytelling. They allow authoring of content which is displayed alongside a view into the data."
-          buttonText="Find out more"
-          buttonLink="https://docs.nextstrain.org/en/latest/guides/communicate/narratives-intro.html"
+          id="groups"
+          // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
i wonder if it would be easier/better to disable this rule for all `*.tsx` files in `static-site/.eslintrc.yaml`?

Reply:
Good idea, opened an issue: #959